_docs/it/schema.md

```diff
@@ -1,3 +1,794 @@
+diverso dal `mainCopyrightOwner`.
+
+### Chiave `legal/authorsFile`
+
+* Tipo: stringa (percorso al file)
+* Presenza: opzionale
+* Esempio: `"doc/AUTHORS.txt"`
+
+Qualche software open-source adotta una convenzione che identifica il
+proprietario del copyright attraverso un file elencante tutte le entità che
+possiedono il copyright. Questo è comune nei progetti fortemente sostenuti
+dalla community ove esistono diversi contributori esterni e non c'è un chiaro
+singolo possessore del copyright. In questi casi, questa chiave può essere
+usata per riferirsi al suddetto file degli autori, usando un percorso relativo alla
+radice (root) del repository. 
+
+
+## Section `maintenance`
+
+Questa sezione fornisce informazioni sullo stato di manutenzione del software,
+utile per valutare se il software è attivamente sviluppato o meno. 
+
+### Chiave `maintenance/type`
+
+* Tipo: enumerate
+* Presenza: obbligatoria
+* Valori: `"internal"`, `"contract"`, `"community"`, `"none"`
+
+Questa chiave descrive come il software è attualmente manutenuto. 
+* `"internal"` significa che il software è manutenuto internamente dal
+  proprietario del repository; 
+* `"contract"` significa che c'è un contratto commerciale che lega un'entità
+  alla manutenzione del software;
+* `"community"` implica che il software è attualmente manutenuto da una o più
+  persone che offrono il loro tempo al progetto;
+* `"none"` significa che il software non è al momento manutenuto.
+
+### Chiave `maintenance/contractors`
+
+* Tipo: array di Contractor (vedi sotto)
+* Presenza: obbligatoria (se `maintainance/type` è `contract`)
+
+Questa chiave descrive l'entità o le entità, se ce ne sono, che attualmente
+hanno un contratto di manutenzione del software. Queste possono essere
+compagnie, organizzazioni o altri nomi collettivi. 
+
+### Chiave `maintenance/contacts`
+
+* Tipo: Lista di Contatti (vedi sotto)
+* Presenza: obbligatoria
+
+Uno o più contatti a chi sta mantenendo il software. 
+
+Questa chiave descrive le persone tecniche che attualmente sono responsabili
+della manutenzione del software. Tutti i contatti devono essere di una persona
+fisica, non una compagnia o un'organizzazione. Se qualcuno funge da
+rappresentante di un'istituzione, deve essere elencato tra `affiliation` dei
+contatti. 
+
+Nel caso di un accordo commerciale (o una catena di tali accordi), specificare
+le entità finali che sono effettivamente contrattate per fornire la
+manutenzione. Non specificare il proprietario del software a meno che sia
+tecnicamente coinvolto anche nella manutenzione del prodotto. 
+
+## Section `localisation`
+
+Questa sezione fornice una panoramica sulle funzionalità di localizzazione del
+software. 
+
+## Chiave `localisation/localisationReady`
+
+* Tipo: booleano
+* Presenza: obbligatoria
+
+Se `yes`, il software ha l'infrastruttura o è stato progettato per essere
+multi-lingua. Non deve essere disponibile in più di una lingua. 
```

Comment:
Non mi è chiaro il significato della frase "non deve essere disponibile in più di una lingua". Che vuol dire? Se abbiamo un software bilingue, `localisationReady` va impostato a `no`?

Reply:
In effetti non si capisce. 
Dovrebbe voler dire che impostare il campo a yes non pregiudica l'esistenza ad oggi di una nuova lingua ma solo dell'infrastruttura necessaria per ospitare una nuova lingua.

Reply:
Ho messo: 'Se `yes`, il software ha l'infrastruttura o è stato progettato per essere
multi-lingua. Ad ogni modo, questo campo non pregiudica l'esistenza di
una traduzione in altre lingue ma si riferisce esclusivamente all'aspetto
tecnologico. Per l'elenco delle lingue disponibili si veda la chiave
`localisation\availableLanguages'.